tests_requre/README.md

```diff
@@ -0,0 +1,49 @@
+# Requre tests
```

Comment:
I'd move this text to CONTRIBUTING.md so that we store all the dev-related content in one place